common/buildcraft/transport/container/ContainerFilteredBuffer.java

```diff
@@ -0,0 +1,52 @@
+/* Copyright (c) 2016 AlexIIL and the BuildCraft team
+ * 
+ * This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not
+ * distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+package buildcraft.transport.container;
+
+import buildcraft.core.lib.gui.slots.SlotBase;
+import buildcraft.core.lib.gui.slots.SlotPhantom;
+import buildcraft.lib.client.sprite.SpriteHolderRegistry;
+import buildcraft.lib.gui.ContainerBCTile;
+import buildcraft.lib.misc.StackUtil;
+import buildcraft.transport.tile.TileFilteredBuffer;
+import net.minecraft.client.renderer.texture.TextureAtlasSprite;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+
+public class ContainerFilteredBuffer extends ContainerBCTile<TileFilteredBuffer> {
+    private static final int PLAYER_INV_START = 86;
+    private static SpriteHolderRegistry.SpriteHolder emptyFilterSlotHolder;
+
+    public static void preInit() {
+        emptyFilterSlotHolder = SpriteHolderRegistry.getHolder("buildcrafttransport:gui/empty_filtered_buffer_slot");
```

Comment:
You should probably move this to a client-only class as sprites don't exist on the server (so this will probably crash when you start up a dedicated server).

I think it would be better to have a dedicated class for transport sprites (as there are quite a few of them).

You can also just use
````java
public static final SpriteHolder SPRITE_THING = SpriteHolderRegistry.getHolder("location:of/thing");
````
Provided you have a method *somewhere* in that class for preInit, which is called by the client proxy.